test/expected/drop_hypertable.out

```diff
@@ -23,6 +23,27 @@ SELECT create_hypertable('should_drop', 'time');
  
 (1 row)
 
+CREATE TABLE hyper_with_dependencies (time timestamp, temp float8);
+SELECT create_hypertable('hyper_with_dependencies', 'time');
+ create_hypertable 
+-------------------
+ 
+(1 row)
+
+CREATE VIEW dependent_view AS SELECT * FROM hyper_with_dependencies;
```

Comment:
I am actually more concerned about the case when you have a reference to a chunk (e.g., a view on a chunk). I guess creating a view on the main table works simply because it is unaware of the fact that it is a hypertable so the view only references the main table and is handled by the regular process utility. 

If you create a test with a view on a chunk, I suspect that `DROP TABLE` will fail because you drop chunks in `drop_hypertable()` without `CASCADE`, even if you gave `CASCADE` in the drop command.